CMake/resolve_dependency_modules/cpr.cmake

```diff
@@ -20,6 +20,9 @@ set(VELOX_CPR_SOURCE_URL
     "https://github.com/libcpr/cpr/archive/refs/tags/${VELOX_CPR_VERSION}.tar.gz"
 )
 
+set(curl_SOURCE BUNDLED)
```

Comment:
:+1: placing that in the cpr file! could you add a comment explaining what this does?

Reply:
OK.